packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -107,15 +110,14 @@ export default createConnector({
           isRefined: v.isRefined,
         }));
 
-    const sortedItems = !isFromSearch && props.searchForFacetValues ?
+    const sortedItems = !isFromSearch && withSearchBox ?
```

Comment:
If the `withSearchBox` is `false` then `isFromSearch` should always be `true`, right? In that case, we could put `withSearchBox` first, no?